### PR TITLE
Feature phone mask

### LINKED ISSUE #336: Change phone mask

### PR DESCRIPTION
**Pull request by idelcano**
closes #336

### CODE REVIEW

**Comment by adrianq** [comment]
@idelcano Change VariantPhoneMask to PhoneMask for the two variants

**Comment by idelcano** [comment]
it's done.

File: app/src/cambodia/java/utils/PhoneMask.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+
+
+    /**
+     * Formatted telephone mask: 0NN NNN NNN{N}
+     */
+    public static final String FORMATTED_PHONENUMBER_MASK = "0\\d{2} \\d{3} \\d{3,4}";
+
+    /**
+     * Formatted telephone mask: 0NN NNN NNN{N}
```
Comment (adrianq) [inline app/src/cambodia/java/utils/PhoneMask.java]: @idelcano Can you change comments in method headers? I got the impression some of them where just copy & paste, maybe I am wrong though

File: app/src/laos/java/utils/PhoneMask.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    public static final String PLAIN_PHONENUMBER_MASK = "(((020)\\d{8})|((030)\\d{7})|\\d{9})";
+
+    /**
+     * Formats number according to mask (xxx) xxx-xxx = 9 (020) xxxx-xxxx = 11  (030) xxx-xxxx = 10
+     * @param phoneValue
+     * @return
+     */
+    public static String formatPhoneNumber(String phoneValue) {
+        //Empty -> nothing to format
+        if (phoneValue == null || "".equals(phoneValue)) {
```
Comment (adrianq) [inline app/src/laos/java/utils/PhoneMask.java]: @idelcano Do we need this `"".equals(phoneValue)`? Probably yes but I do not understand why...

**Comment by adrianq** [comment]
@idelcano A couple of minor comments before merging it...

File: app/src/laos/java/utils/PhoneMask.java (hunk 2 above)
Comment (idelcano) [inline app/src/laos/java/utils/PhoneMask.java]: I do not understand why is there, I removed it.

**Comment by idelcano** [comment]
its fixed!

**Comment by adrianq** [comment]
@idelcano I have just merged it however I am little bit worried about this bit  `"".equals(phoneValue)` I got the impression there was a reason for it. Once I am done with all the PRs, let's do some testing with the phone field (no phone, a blank space, etc.)

**Pull request merged**